setup.cfg

```diff
@@ -61,6 +61,8 @@ test =
 docs =
     sphinx-rtd-theme
     sphinx-astropy
+roman =
```

Comment:
if you haven't already, add a few lines to the readme and install instructions showing how to install with the optional roman stuff

Reply:
also, im not sure how this works but if you define this additional section here, will something like pip install .['roman'] work automatically? i see that you've defined the 'romandeps' option, but is that redundant?

Reply:
`romandeps` uses tox syntax, and tox is what is being used by the CI. What we have is copied over from how astropy do things.

If you want to run the test separately without tox, do you have to do this manually (no quotes):

`pip install -e .[roman]`

This syntax is standard in Python, not specific to Roman.

Also see https://github.com/spacetelescope/jdaviz/pull/1822#discussion_r1166870021

Reply:
Right. With this PR, you can do
```
pip install jdaviz[roman]
pip install -e .[roman]
```
To get the roman deps. Before this PR, `[doc]` was already supported.

Specifying `-romandeps` in the tox args makes sure the correct packages are installed in the tox env for that job.

Reply:
Right, basically romandeps tells tox to go install the roman optional dependencies but using tox syntax. Or am I misunderstanding your comment?